prometheus-exporters/openstack-exporter/alerts/cinder.alerts

```diff
@@ -1,50 +1,67 @@
+      summary: 'Cinder backend {{ $labels.backend }}, shard {{ $labels.shard }} has 3 or less3 datastores with > 30% free space left'
+  - alert: CinderShardFreeSpaceCritical
+    expr: count by (shard, backend) (sum(cinder_virtual_free_capacity_gib / cinder_available_capacity_gib) by (shard, backend, pool) > .30) < 2
     for: 15m
     labels:
       severity: warning
       tier: os
       service: cinder
-      playbook: docs/support/playbook/cinder/cinder_reserved_percent_low.html
+      playbook: docs/support/playbook/cinder/cinder_low_free_space.html
     annotations:
-      description: 'Cinder backend {{ $labels.backend }}, shard {{ $labels.shard }}'
-      summary: 'Cinder Shard reserved space available count critical'
-  - alert: CinderBackendStorageEmptyCritical
+      description: 'Cinder backend {{ $labels.backend }}, shard {{ $labels.shard }} has less than 2 datastores with > 30% free space left'
+      summary: 'Cinder backend {{ $labels.backend }}, shard {{ $labels.shard }} has less than 2 datastores with > 30% free space left'
+  - alert: CinderShardMaxVolumeSizeWarning
     expr: >
-      sum(cinder_free_capacity_gib == 0) by (region, shard, backend) or sum(cinder_total_capacity_gib == 0) by (region, shard, backend)
+      count by (shard, backend) (sum(cinder_available_capacity_gib) by (shard, backend, pool)) - count by (shard, backend) (sum by(shard, backend, pool) (cinder_virtual_free_capacity_gib) - sum by(shard, backend, pool) (cinder_per_volume_gigabytes) <=0) <= 3
     for: 15m
     labels:
-      severity: critical
+      severity: warning
       tier: vmware
       service: cinder
       context: "openstack-exporter"
-      meta: "Cinder backend {{ $labels.shard }}/{{ $labels.backend }} has no storage capacity."
-      playbook: docs/support/playbook/cinder/cinder_storage_profile_empty.html
+      meta: "Cinder backend {{ $labels.shard }}/{{ $labels.backend }} has 3 or less datastores that can accept max volume size"
+      playbook: docs/support/playbook/cinder/cinder_low_free_space.html
     annotations:
-      description: "Cinder backend {{ $labels.shard }}/{{ $labels.backend }} has no storage capacity."
-      summary: "Cinder backend {{ $labels.shard }}/{{ $labels.backend }} has no storage capacity."
+      description: "Cinder backend {{ $labels.shard }}/{{ $labels.backend }} has 3 or less datastores that can accept max volume size"
+      summary: "Cinder backend {{ $labels.shard }}/{{ $labels.backend }} has 3 or less datastores that can accept max volume size"
+  - alert: CinderShardMaxVolumeSizeCritical
+    expr: >
+      count by (shard, backend) (sum(cinder_available_capacity_gib) by (shard, backend, pool)) - count by (shard, backend) (sum by(shard, backend, pool) (cinder_virtual_free_capacity_gib) - sum by(shard, backend, pool) (cinder_per_volume_gigabytes) <=0) < 2
+    for: 15m
+    labels:
+      severity: warning
```

Comment:
This is warning, but named crititical. Is that correct like that?

Reply:
yes because I thought that ops had to approve something being moved to critical first?